ocaml/nbd/src/main.ml

```diff
@@ -255,4 +255,9 @@ let () =
      in the failed state, and a backtrace will show up in the logs. *)
   Cleanup.Runtime.register_signal_handler () ;
   setup_logging () ;
+  let handler exn _ =
+    Lwt_log.ign_error (Printexc.to_string exn) ;
+    Lwt_log.ign_error (Printexc.get_backtrace ())
```

Comment:
Could the logging call destroy the backtrace? I think it would be best to let-bind the backtrace before doing anything else in the handler.

Reply:
Not sure on that one, I can do a let bind to ensure that

Reply:
The handler does have a backtrace on it I am no using to print the backtrace.